src/documentation/ToolkitDocumentation/ToolkitBreadcrumbHeading.tsx

```diff
@@ -54,14 +73,29 @@ const ToolkitBreadcrumbHeading = ({
           </Text>
         </Text>
       </Button>
-      <Text
-        as="h2"
-        fontSize="3xl"
-        fontWeight="semibold"
-        fontFamily={titleFontFamily}
-      >
-        {title}
-      </Text>
+      <HStack align="flex-start" spacing={4}>
+        {icon && (
+          <Image
+            src={imageUrlBuilder.image(icon.asset).width(80).height(64).url()}
```

Comment:
I think this is pointless because we know the icons are svgs. In fact, we might be able to query the asset url directly from Sanity and avoid using the image builder at all here.